content/docs/tasks/telemetry/distributed-tracing/index.md

```diff
@@ -137,6 +137,35 @@ public Response bookReviewsById(@PathParam("productId") int productId,
 
 When you make downstream calls in your applications, make sure to include these headers.
 
+## Trace Sampling
+
+Istio captures a trace for all requests by default. Every time you
```

Comment:
You need to preface the second sentence with something like:

For example, when using the Bookinfo sample application, ...